<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LSB文字提取</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/m-水印.png</normaloff>images/m-水印.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>321</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="fileRoadSelect"/>
    </item>
    <item>
     <widget class="QPushButton" name="imageSelectBtn">
      <property name="text">
       <string>选择图片</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/素材.png</normaloff>images/素材.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>301</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLineEdit" name="outfileRoadText"/>
    </item>
    <item>
     <widget class="QPushButton" name="outfileRoadBtn">
      <property name="text">
       <string>选择输出目录</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/open.png</normaloff>images/open.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="compressBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>提取信息</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/水印.png</normaloff>images/水印.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入提取长度</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
